app-tasks/rf/src/rf/uploads/geotiff/create_thumbnails.py

```diff
@@ -0,0 +1,121 @@
+"""Creates thumbnails for geotiff imagery
+
+Depends on having geotiff file already downloaded
+"""
+import os
+import subprocess
+
+import boto3
+
+from rf.models import Thumbnail
+
+from .io import get_geotiff_dimensions
+
+def create_thumbnails(tif_path, scene_id, organization_id):
+    """Creates thumbnails based on a geotiff path and scene
+
+    Args:
+        tif_path (str): path of local tiff file
+        scene_id (str): uuid of the scene
+
+    Returns:
+        List[Thumbnail]
+    """
+
+    r_tif_path = 'resampled_%s' % tif_path
+    rp_tif_path = 'reprojected_%s' % tif_path
+
+    dim = get_geotiff_dimensions(tif_path)
+    max_dim = float(max(get_geotiff_dimensions(tif_path)))
+
+    size_large = 1024
+    size_small = 256
+
+    scale_large = size_large / max_dim
+    scale_small = size_small / max_dim
+
+    dim_large = (dim[0] * scale_large, dim[1] * scale_large)
+    dim_small = (dim[0] * scale_small, dim[1] * scale_small)
+
+    # Create paths for each size thumb
+    path_large = '%s-LARGE.png' % scene_id
+    path_small = '%s-SMALL.png' % scene_id
+
+    # Create urls for each size Thumbnail
+    url_large = '/thumbnails/%s' % path_large
+    url_small = '/thumbnails/%s' % path_small
+
+    try_to_remove([r_tif_path, rp_tif_path, path_large, path_small])
+
+    if os.path.isfile(tif_path):
+        # Resample tif
+        subprocess.call([
+            'gdalwarp', tif_path, r_tif_path,
+            '-ts', str(dim_large[0]), str(dim_large[1]),
+            '-q'
+        ])
+
+        # Reproject tif
+        subprocess.call([
+            'gdalwarp', r_tif_path, rp_tif_path,
+            '-t_srs', 'EPSG:3857',
+            '-q'
+        ])
+
+        # Convert tif to pngs (large)
+        subprocess.call([
+            'gdal_translate', rp_tif_path, path_large,
+            '-outsize', str(dim_large[0]), str(dim_large[1]),
+            '-of', 'PNG',
+            '-q'
```

Comment:
The NIR in the Kayak GeoTIFFs is getting turned into an alpha channel in the resulting PNG, which causes them to have some weird transparency effects:
![thumbnailtrans](https://cloud.githubusercontent.com/assets/447977/22571183/b3ce3d3e-e96c-11e6-9681-8cc86ba338b6.png)

Changing the call to the following fixed the problem, but it won't work for a TIFF where the RGB bands aren't the first three, so we may need to revisit this once we add another data provider.
```python
          subprocess.call([                                                                              
              'gdal_translate', rp_tif_path, path_large,                                                 
              '-b', '1',                                                                                 
              '-b', '2',                                                                                 
              '-b', '3',                                                                                 
              '-outsize', str(dim_large[0]), str(dim_large[1]),                                          
              '-of', 'PNG',                                                                              
              '-q'                                                                                       
          ])                                                                                             

```